examples/onsasExample_cantileverPendulum.m

```diff
@@ -6,15 +6,15 @@
 % ----------------------------------------------------------------------
 %% Geometric Properties
 %Geometric properties truss:
-Et = 1e9 ; dt = .05; At = pi*dt^2/4 ;  Lt = 1 ; nut = 0.3 ;  rhot = 1 ; 
+Et = 10e11 ; At = .1 ; dt = sqrt(4*At/pi);   Lt = 3.0443 ; nut = 0.3 ;  rhot = 65.6965 ; 
```

Comment:
there might be a small mistake here, is At switched with dt?
```matlab
At = .1 ; dt = sqrt(4*At/pi);
```

Reply:
This is associated whith the simplePendulum example form Crisfield. He define an area and then i calculated de tha diamater for that area.